react/ReadMore/ReadMore.js

```diff
@@ -135,15 +135,17 @@ class ReadMore extends PureComponent<Props, State> {
           ) : null}
         </div>
         {showMoreLessButton && (
-          <Text
+          <TextLink
             strong
+            color="transparent"
+            type="button"
             chevron={showMore ? 'up' : 'down'}
             className={styles.showMore}
-            component={TextLink}
+            component={Button}
```

Comment:
Confirming that the desired effect is that this render a `button` rather than an `anchor` element? Also surprised that `type="button"` wasn't the default in our `Button` component — learn something new everyday.

Reply:
So what I didn't realise here was that the hit area spanned the entire width of the text previously. My change here has made the More link button shaped which I thought was an improvement but actually makes the hit area smaller. Going to roll this back and manually apply the font-weight change

![Screen Shot 2019-04-04 at 12 02 30 pm](https://user-images.githubusercontent.com/1896277/55522660-99343580-56d1-11e9-8ce1-72dc667179b6.png)